docs/createInputsReducer.md

```diff
@@ -0,0 +1,62 @@
+## createInputsReducer(inputsConfig:Object)
+
+You use this create reducers for your forms. You need one reducer per form. 
+
+Example:
+
+    import { createStore, combineReducers } from 'redux';
+    import { reducer as formReducer } from 'redux-form';
+    
+    const inputsConfig = {
+        ... redux-inputs configuration ...
+    };
+    
+    const reducers = {
+        inputs: createInputsReducer(inputsConfig)
+    };
+    
+    const reducer = combineReducers(reducers);
+    const store = createStore(reducer);
+    
+### Use a different redux mount point
```

Comment:
Make it more clear that this is optional. In most cases the user shouldn't have to worry about this at all.